firestore/integration_test_internal/src/query_test.cc

```diff
@@ -236,11 +236,17 @@ TEST_F(QueryTest, TestKeyOrderIsDescendingForDescendingInequality) {
                   {"e", {{"foo", FieldValue::Double(21.0)}}},
                   {"f", {{"foo", FieldValue::Integer(66)}}},
                   {"g", {{"foo", FieldValue::Double(66.0)}}}});
-  QuerySnapshot snapshot = ReadDocuments(
+  const Query& query =
       collection.WhereGreaterThan("foo", FieldValue::Integer(21))
-          .OrderBy(FieldPath({"foo"}), Query::Direction::kDescending));
+          .OrderBy(FieldPath({"foo"}), Query::Direction::kDescending);
+  QuerySnapshot snapshot = ReadDocuments(query);
   EXPECT_EQ(std::vector<std::string>({"g", "f", "c", "b", "a"}),
             QuerySnapshotToIds(snapshot));
+
+  const AggregateQuery& aggregate_query = query.Count();
```

Comment:
Adding count tests to all existing tests isn't a great idea because it's expanding each test to test two things: the original query and the aggregate query. This also doesn't scale for future aggregations. If you really want this coverage, please copy these tests into aggregate_query_test.cc or maybe a count_query_test.cc.